wayat/frontend/lib/features/groups/pages/manage_group_page.dart

```diff
@@ -14,13 +15,23 @@ import 'package:wayat/features/groups/controllers/groups_controller/groups_contr
 import 'package:wayat/features/groups/controllers/manage_group_controller/manage_group_controller.dart';
 import 'package:wayat/features/groups/widgets/create_group_contact_tile.dart';
 import 'package:wayat/lang/app_localizations.dart';
+import 'package:wayat/services/common/platform/platform_service_libw.dart';
 
 /// Page that provides functionality to edit and create new [Group] entities.
 class ManageGroupPage extends StatelessWidget {
   final ManageGroupController controller;
-  ManageGroupPage({ManageGroupController? controller, Group? group, Key? key})
-      : controller = controller ?? ManageGroupController(group: group),
-        super(key: key);
+
+  final PlatformService platformService;
+  
+  ManageGroupPage({
+    ManageGroupController? controller, 
+    PlatformService? platformService,
+    Group? group, 
+    Key? key
+  }) : 
```

Comment:
Please, note that this is not the format define by flutter. If you want to have the }) you need to put a , before that